tests/test_wallet.py

```diff
@@ -280,3 +291,116 @@ def test_from_int(self):
 
     def test_repr(self):
         assert repr(PrivateKeyTestnet(WALLET_FORMAT_MAIN)) == '<PrivateKeyTestnet: bchtest:qzfyvx77v2pmgc0vulwlfkl3uzjgh5gnmqjxnsx26x>'
+
+
+class TestPrivateKeyRegtest:
+    def test_init_default(self):
+        private_key = PrivateKeyRegtest()
+
+        assert private_key._address is None
+        assert private_key.balance == 0
+        assert private_key.unspents == []
+        assert private_key.transactions == []
+
+    def test_address(self):
+        private_key = PrivateKeyRegtest(WALLET_FORMAT_REGTEST)
+        assert private_key.address == BITCOIN_CASHADDRESS_REGTEST
+
+    def test_to_wif(self):
+        private_key = PrivateKeyRegtest(WALLET_FORMAT_REGTEST)
+        assert private_key.to_wif() == WALLET_FORMAT_REGTEST
+
+        private_key = PrivateKeyRegtest(WALLET_FORMAT_COMPRESSED_REGTEST)
+        assert private_key.to_wif() == WALLET_FORMAT_COMPRESSED_REGTEST
+
+    def test_get_balance(self):
+        private_key = PrivateKeyRegtest(WALLET_FORMAT_REGTEST)
+        balance = int(private_key.get_balance())
+        assert balance == private_key.balance
+
+    def test_get_unspent(self):
+        private_key = PrivateKeyRegtest(WALLET_FORMAT_REGTEST)
+        unspent = private_key.get_unspents()
+        assert unspent == private_key.unspents
+
+    def test_get_transactions(self):
+        private_key = PrivateKeyRegtest(WALLET_FORMAT_REGTEST)
+        transactions = private_key.get_transactions()
+        assert transactions == private_key.transactions
+
+    #This tests requires the local node to be continuously generating blocks
+    #marking 'skip' until auto-block generation is functional
```

Comment:
Maybe this comment can be inside the function?  
Also please add a white space after the `#` 😇

Reply:
Done [https://github.com/pybitcash/bitcash/pull/82/commits/3dce8a00388e28f58d23367afb1906a56680efe9](https://github.com/pybitcash/bitcash/pull/82/commits/3dce8a00388e28f58d23367afb1906a56680efe9)